### PR TITLE
updates to the project docs naming and organization + wrote new project setup instructions for dbt cloud

### LINKED ISSUE #275: Add dbt project setup to DSE docs
add a step about adding "transform" to project subdirectory

Image

[Project Setup](https://cagov.github.io/data-infrastructure/setup/) can be change to "local env setup"

"dbt cloud" section of [this hand off checklist](https://docs.google.com/spreadsheets/d/1YRBzOH9wS-CgqCq6P8aPNy6Yu-KjHigOPmweAcmX79s/edit#gid=0) could be useful too

### PR DESCRIPTION
**Pull request by britt-allen**
fixes #275

### CODE REVIEW

File: docs/dbt.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+
+To set up a new project on dbt Cloud follow these steps:
+
+1. Give your new project a name.
+1. Click *Advanced settings* and in the *Project subdirectory* field, enter "transform"
+1. Select a datawarehouse connection. (e.g. Snowflake, BigQuery, Redshift)
+1. For the *Development credentials* section you'll want to:
+    1. Under *Auth method* select *Key pair*
+    1. Enter your data warehouse username
+    1. Enter the private key amd private keypassphrase
```
Comment (JamesSLogan) [inline docs/dbt.md:13]: "amd private keypassphrase" -> "and private key passphrase"

File: docs/dbt.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+    1. For more guidance, read [dbt's docs on connecting to Snowflake via key pair](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-snowflake#key-pair)
+1. Finally click the *Test Connection* button.
+1. Connect the apprioriate repository (usually GitHub). Read [dbt's docs on connecting to GitHub](https://docs.getdbt.com/docs/cloud/git/connect-github).
```
Comment (JamesSLogan) [inline docs/dbt.md:16]: apprioriate -> appropriate

File: docs/dbt.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
 # dbt on the Data Services and Engineering team
 
+## New project setup
+
+To set up a new project on dbt Cloud follow these steps:
+
+1. Give your new project a name.
+1. Click *Advanced settings* and in the *Project subdirectory* field, enter "transform"
+1. Select a datawarehouse connection. (e.g. Snowflake, BigQuery, Redshift)
```
Comment (JamesSLogan) [inline docs/dbt.md:9]: data warehouse

File: docs/dbt.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+    1. Enter your data warehouse username
+    1. Enter the private key amd private keypassphrase
+    1. For more guidance, read [dbt's docs on connecting to Snowflake via key pair](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-snowflake#key-pair)
+1. Finally click the *Test Connection* button.
+1. Connect the apprioriate repository (usually GitHub). Read [dbt's docs on connecting to GitHub](https://docs.getdbt.com/docs/cloud/git/connect-github).
+
+Once you're through the first five steps you can return to the dbt homepage and click the Settings button in the upper right corner. From there you can follow the steps to configure three environments for Continuous intergation - CI, development, and production. Read [dbt's docs on CI in dbt Cloud](https://docs.getdbt.com/docs/deploy/continuous-integration). Read [dbt's docs on creating production (deployment) environments](https://docs.getdbt.com/docs/deploy/deploy-environments) and [dbt's docs on creating and scheduling deploy jobs](https://docs.getdbt.com/docs/deploy/deploy-jobs#create-and-schedule-jobs).
+
+You'll also want to [configure notifications for job failures](Configure notifications for job failures).
```
Comment (JamesSLogan) [inline docs/dbt.md:20]: I think this link needs a corrected URL

**Review by JamesSLogan — COMMENTED** [review]
Thanks Britt, the new documentation looks good with a few small corrections needed. I am getting 404s for the following:
`images/environment_variables.png`
`local-setup.md`
`snowflake-project-setup.md`

File: docs/dbt.md (hunk 4 above)
Comment (britt-allen) [reply docs/dbt.md:20]: which of the 3 links are you referring to?

Comment (britt-allen) [reply docs/dbt.md:20]: I just went to all 3 URLs and they work for me.

Comment (JamesSLogan) [reply docs/dbt.md:20]: The one on line 20:
```
[configure notifications for job failures](Configure notifications for job failures)
```

**Comment by JamesSLogan** [comment]
> I am not sure why, it renders for me locally with no problem.

If the files are present on your local branch but not github, is there any chance they aren't getting committed?

File: docs/dbt.md (hunk 4 above)
Comment (britt-allen) [reply docs/dbt.md:20]: Thank you!

**Comment by britt-allen** [comment]
Thanks for review and feedback @JamesSLogan! I think this is ready now

**Comment by JamesSLogan** [comment]
Almost there - `mkdocs` now shows me these messages:
```
WARNING -  Doc file 'code-review.md' contains a relative link './setup.md#install-dependencies', but the target 'setup.md' is not found among documentation files.
WARNING -  Doc file 'snowflake-project-setup.md' contains a relative link './setup.md#aws', but the target 'setup.md' is not found among documentation files.
```
I believe these 2 links need to be to `local-setup` instead of `setup` now.

**Review by JamesSLogan — APPROVED** [review]
Looks great

**Comment by britt-allen** [comment]
Thanks for catching and flagging those warnings @JamesSLogan and for all the feedback and review.

**Comment by ian-r-rose** [comment]
Nice work @britt-allen!

**Comment by britt-allen** [comment]
Thanks @ian-r-rose!

**Pull request merged**